doc/CMakeLists.txt

```diff
@@ -282,29 +273,28 @@ doxygen_add_docs(doxygen_xml
 #-----------------------------------------------------------------------------#
 # MMD PROCESSING
 
-# Note that *all* mermaid files must be present at the same time to prevent
+# Note that *all* dot files must be present at the same time to prevent
 # errors when doing separate HTML/PDF builds: the doctrees directory caches the
 # file listing and will fail if only one image type is generated
-set(_mermaid_output)
-set(_png_fallback "${CMAKE_CURRENT_SOURCE_DIR}/_static/mermaid-not-found.png")
-set(_output_base "${CMAKE_CURRENT_SOURCE_DIR}/_static/mermaid")
+set(_dot_output)
+set(_png_fallback "${CMAKE_CURRENT_SOURCE_DIR}/_static/dot-not-found.png")
+set(_output_base "${CMAKE_CURRENT_SOURCE_DIR}/_static/dot")
 
-macro(_celer_build_mermaid inp)
+macro(_celer_build_dot inp)
   get_filename_component(basename "${inp}" NAME_WE)
-  if(MMDC_EXECUTABLE)
-    # Generate SVG and PDF for HTML, PDF respectively
-    foreach(_fmt pdf svg)
+  if(DOXYGEN_DOT_EXECUTABLE)
```

Comment:
(minor) Why the link between `Doxygen` and `Dot`?

Reply:
CMake's FindDoxygen automatically finds `dot`, and it includes some extra logic about where to look; so we might as well piggyback off it.